docs/src/language_servers.md

```diff
@@ -751,6 +752,33 @@ See [Javascript/TypeScript](#javascripttypescript).
     }
     ```
 
+5. Optional: to enable some useful commands provided by language server, add the following to `*.sublime-commands` (see more [here](commands.md#execute-server-commands)):
+
+    <!-- how to call: see https://github.com/nvarner/typst-lsp/blob/master/editors/vscode/src/extension.ts -->
+    ```jsonc title="Packages/User/Default.sublime-commands"
+    [
+        // ...
+        {
+            "caption": "typst-lsp - Pin the main file to the currently openning document",
+            "command": "lsp_execute",
+            "args": {
+                "session_name": "typst-lsp",
+                "command_name": "typst-lsp.doPinMain",
+                "command_args": ["${file_uri}"]
+            }
+        },
+        {
+            "caption": "typst-lsp - Unpin the main file",
+            "command": "lsp_execute",
+            "args": {
+                "session_name": "typst-lsp",
+                "command_name": "typst-lsp.doPinMain",
+                "command_args": ["detached"]
+            }
+        },
```

Comment:
I don't really understand this functionality but it looks like in VSCode this command unpins and then pins the current file. To be fair, based on the name, I would expect it to just unpin but still weird to see this behavior there.

Reply:
> unpins and then pins the current file

Probably you're right, I missed that, but I will check

Reply:
Ok, I'm not sure what's really going on, but it should work, at least in the same way as in the vscode extension